.travis.yml

```diff
@@ -2,6 +2,10 @@ sudo: required
 dist: trusty
 language: scala
 
+env:
+  - MVN_FLAG=
+  - MVN_FLAG=-Dminicluster-2.10.spark.version=2.0.1 -Dminicluster-2.11.spark.version=1.6.2
```

Comment:
Official version is tested by line 6.
The default for minicluster-2.10.spark.version is 1.6.2 and inicluster-2.11.spark.version is 2.0.1